build/win/makedist_win.bat

```diff
@@ -46,7 +46,7 @@ REM Arno: When adding files here, make sure tribler.nsi actually
 REM packs them in the installer .EXE
 
 python3 -m pip install --upgrade --no-deps --force-reinstall pydantic --no-binary pydantic
-python3 -m pip install --upgrade --no-deps --force-reinstall typing_extensions --no-binary typing_extensions
+python3 -m pip install --upgrade --no-deps --force-reinstall typing_extensions==3.10.0.2 --no-binary typing_extensions
```

Comment:
Without --no-binary PyInstaller does not include this module into the Tribler exe file on Windows
The latest version of typing_extension (4.0.0) is currently not working as well.
After #6534 we can move it into requirements with --no-binary specification